test/TestImportWithMalformed.py

```diff
@@ -0,0 +1,59 @@
+from collections import namedtuple
+import os
+
+import pytest
+
+from ansiblelint import Runner, RulesCollection
+
+
+PlayFile = namedtuple('PlayFile', ['name', 'content'])
+
+
+IMPORT_TASKS_MAIN = PlayFile('import-tasks-main.yml', '''
+- oops this is invalid
+''')
+
+PLAY_IMPORT_TASKS = PlayFile('playbook.yml', '''
+- hosts: all
+  tasks:
+    - import_tasks: import-tasks-main.yml
+''')
+
+
+@pytest.fixture
+def play_file_path(tmp_path):
+    p = tmp_path / 'playbook.yml'
+    return str(p)
+
+
+@pytest.fixture
+def rules():
+    rulesdir = os.path.join('lib', 'ansiblelint', 'rules')
```

Comment:
Let's use pathlib instead
```suggestion
    rulesdir = str(Path('lib') / 'ansiblelint' / 'rules')
```